modules/dmrpp_module/data/get_dmrpp.in

```diff
@@ -115,50 +116,54 @@ BES.DefaultResponseMethod=POST
 # FONc handler specific parameters:
 #-----------------------------------------------------------------------#
 #Make sure this follows the netCDF-4 enhanced model
-FONc.ClassicModel=false
+#jhrg FONc.ClassicModel=false
 #Don't generate global attributes since we only
 #care about the variable value information.
-FONc.NoGlobalAttrs=true
+#jhrg FONc.NoGlobalAttrs=true
 
 #-----------------------------------------------------------------------#
 # HDF5 handler specific parameters:
 #-----------------------------------------------------------------------#
 # EnableCF: Groom the HDF5 data to follow the CF conventions
 #   (true,yes|false,no, defaults to false)
 #
+# HDF5 handler keys that are not the compiled-in defaults. jhrg 5/24/22
+H5.EnableCheckNameClashing=true
```

Comment:
THis is likely to produce different output and new issues for people that have already used get_dmrpp to build dmr++ files. Has this been tested widely? I think this is a dangerous change.

Reply:
I didn't change the value, I moved the keys that are not just reiterating the default values to one place. The fact that it's clearly not obvious what is a default and what is non-default is exactly why I did this!

Reply:
ok